File: C#/1/1_2.cs
using System;

class Program
{
    static void Main()
    {
        int[] arr = { 5, 10, 15, 20, 25 };
        printArray(arr);
    }

    static void printArray(int[] a)
    {
        for (int i = 0; i < a.Length; i++)
        {
            Console.WriteLine("Value at index " + i + ": " + a[i]);
        }
    }
}

File: JavaScript/1/1_2.js
let arr = [5, 10, 15, 20, 25];
function listArray(selArr) {
    for (let i = 0; i < selArr.length; i++) {
        console.log("Value at index " + i + ": " + selArr[i]); // vypisuje pořadí dokud nedojde na konec
    }
}
listArray(arr);
